promptsource/templates/app_reviews/templates.yaml

```diff
@@ -58,56 +46,44 @@ templates:
     name: generate_review
     reference: Generate a review from the rating.
   9746ce4b-ac58-4dfb-9783-d77c95cb62cf: !Template
-    answer_choices: null
-    answer_choices_key: null
+    answer_choices:
+    - "\u2605"
+    - "\u2605\u2605"
+    - "\u2605\u2605\u2605"
+    - "\u2605\u2605\u2605\u2605"
+    - "\u2605\u2605\u2605\u2605\u2605"
+    answer_choices_key: star
     id: 9746ce4b-ac58-4dfb-9783-d77c95cb62cf
-    jinja: 'What would be the *-rating of this review (* being lowest and ***** being
-      highest) : {{review}}?
-
-      |||{% if star==5 %}
-
-      *****
-
-      {% elif star==4%}
-
-      ****
-
-      {% elif star==3%}
-
-      ***
-
-      {% elif star==2%}
-
-      **
-
-      {% else %}
-
-      *
-
-      {% endif %}'
+    jinja: "What would be the \u2605-rating of this review (\u2605 being the lowest\
+      \ and \u2605\u2605\u2605\u2605\u2605 being the highest)? \"{{review}}\"\n|||\n\
+      {{answer_choices[star-1]}}"
     metadata: !TemplateMetadata
       _do_eval: false
       _do_train: true
-      choices_in_prompt: null
-      metrics: []
-      original_task: null
+      choices_in_prompt: false
+      metrics:
+      - Accuracy
+      - Spearman Correlation
+      original_task: false
     name: convert_to_star_rating
     reference: Given the review, generate a star rating.
   d34e1413-2699-4701-baa2-05d931d012ba: !Template
     answer_choices: null
     answer_choices_key: null
     id: d34e1413-2699-4701-baa2-05d931d012ba
-    jinja: 'How would you rate this review from 1 to 5 (1 being lowest and 5 being
-      highest): {{review}}?
+    jinja: 'On a scale of 1-5 (with 1 being least favorable and 5 being most favorable),
+      how would you rate this review? "{{review}}"
 
       |||
 
       {{star}}'
     metadata: !TemplateMetadata
       _do_eval: false
       _do_train: true
-      choices_in_prompt: true
-      metrics: []
-      original_task: null
+      choices_in_prompt: false
+      metrics:
+      - Accuracy
+      - Spearman Correlation
+      original_task: false
```

Comment:
Why isn't this the original task?

Reply:
I also wondered and failed to find the full reference.

The paper seems in a different title https://explore.openaire.eu/search/publication?pid=10.5167%2Fuzh-113425 and the server responded 403. (If we're going to be more thorough, I guess I will have to contact the dataset uploader.)
Nevertheless, judging by the abstract, the dataset curators seem using it for categorizing reviews into a custom taxonomy, instead of associating reviews and stars.

Reply:
Thanks! That's helpful.